CMakeLists.txt

```
@@ -256,15 +253,44 @@ write_basic_package_version_file(
 )
 
 # Configure '<PROJECT-NAME>Config.cmake'
+# version to be installed
 # Use variables:
 #   * targets_export_name
 #   * PROJECT_NAME
+# Used by Config.cmake.in
+set(OpenGR_INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include")
+set(OpenGR_INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib")
+set (OpenGR_EXPORT_TARGET_DIR \${CMAKE_CURRENT_LIST_DIR})
```

Great thanks.
Is the `\` a typo ?
Shouldn't it be written as:
```suggestion
set (OpenGR_EXPORT_TARGET_DIR "${CMAKE_CURRENT_LIST_DIR}")
```

---

I added it so that the file generated is the same as before the PR (do no inline the variable).